lua/nvim-tree/utils.lua

```diff
@@ -264,6 +264,48 @@ function M.table_create_missing(tbl, sub)
   return t
 end
 
+-- Move a value from src to dst if value is nil on dst
+-- @param src to copy from
+-- @param src_path dot separated string of sub-tables
+-- @param src_pos value pos
+-- @param dst to copy to
+-- @param dst_path dot separated string of sub-tables, created when missing
+-- @param dst_pos value pos
+function M.move_missing_val(src, src_path, src_pos, dst, dst_path, dst_pos)
+  if
+    not pcall(vim.validate, {
```

Comment:
i think we can deliberaly fail when this function is called with the wrong arguments instead of silently returning ?

Reply:
That works. Are you thinking of a warning message?

Signing off for now... I'll get to this on the weekend.

Reply:
:+1: have a good week !

Reply:
This is a nice pattern we can use more.

`[NvimTree] move_missing_val: dst_path: expected string, got number`